### PR TITLE
IA update Infrastructure > Running a Node > Set Up a Bootnode

### PR DESCRIPTION
**Pull request by dawnkelly09**
This PR does the following:
- moves "Set Up a Bootnode" from old location to new home in IA
- updates folders for code snippet, moves snippet file, updates snippet path in docs

No content changes as this was previously merged.

### CODE REVIEW

**Review by CrackTheCode016 — APPROVED** [review]

File: infrastructure/running-a-node/setup-bootnode.md [hunk 1]
```diff
@@ -1,4 +1,107 @@
...
----
\ No newline at end of file
+description: Learn how to configure and run a bootnode for Polkadot, including P2P, WS, and secure WSS connections with network key management and proxies.
+---
+
+# Set Up a Bootnode
+
+## Introduction
+
+Bootnodes are essential for helping blockchain nodes discover peers and join the network. When a node starts, it needs to find other nodes, and bootnodes provide an initial point of contact. Once connected, a node can expand its peer connections and play its role in the network, like participating as a validator. This guide will walk you through setting up a Polkadot bootnode, configuring P2P, WebSocket (WS), secure WSS connections, and managing network keys. You'll also learn how to test your bootnode to ensure it is running correctly and accessible to other nodes.
```
Comment (eshaben) [inline infrastructure/running-a-node/setup-bootnode.md:10]:
```suggestion
Bootnodes are essential for helping blockchain nodes discover peers and join the network. When a node starts, it needs to find other nodes, and bootnodes provide an initial point of contact. Once connected, a node can expand its peer connections and play its role in the network, like participating as a validator.

This guide will walk you through setting up a Polkadot bootnode, configuring P2P, WebSocket (WS), secure WSS connections, and managing network keys. You'll also learn how to test your bootnode to ensure it is running correctly and accessible to other nodes.
```

File: infrastructure/running-a-node/setup-bootnode.md [hunk 2]
```diff
@@ -1,4 +1,107 @@
...
+Bootnodes must be accessible through three key channels to connect with other nodes in the network:
+
+- **P2P** - a direct peer-to-peer connection, set by:
+
+    ```bash
+
+    --listen-addr /ip4/0.0.0.0/tcp/INSERT_PORT
+
+    ```
+    Note: This is not enabled by default on non-validator nodes like archive RPC nodes
```
Comment (eshaben) [inline infrastructure/running-a-node/setup-bootnode.md:31]:
```suggestion
    !!! note
        This is not enabled by default on non-validator nodes like archive RPC nodes.
```

File: infrastructure/running-a-node/setup-bootnode.md [hunk 3]
```diff
@@ -1,4 +1,107 @@
...
+
+You can create a node key using:
+
+ ``` bash
+ polkadot key generate-node-key
+ ``` 
+ 
+This key can be used in the startup command line.
+
+It is imperative that you backup the node key. If it is included in the `polkadot` binary it is hardcoded into the binary which must be recompiled to change the key.
```
Comment (eshaben) [inline infrastructure/running-a-node/setup-bootnode.md:50]:
```suggestion
It is imperative that you backup the node key. If it is included in the `polkadot` binary, it is hardcoded into the binary, which must be recompiled to change the key.
```

**Review by eshaben — COMMENTED** [review]
Sorry! I know you didn't change this at all, but I noticed a few minor things 😬

**Review by eshaben — APPROVED** [review]
LGTM 🚢

**Pull request merged**